src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs

```diff
@@ -725,8 +718,7 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
                     // Mark socket connected.
                     if (socketError == SocketError.Success)
                     {
-                        if (s_loggingEnabled)
-                            SocketsEventSource.Connected(_currentSocket, _currentSocket.LocalEndPoint, _currentSocket.RemoteEndPoint);
+                        NetEventSource.Connected(_currentSocket, _currentSocket.LocalEndPoint, _currentSocket.RemoteEndPoint);
```

Comment:
No, not really.  As above, it's more of a "what's the general principle?" sort of issue.  I'll try to elaborate better in a subsequent comment.